src/main/scala/com/gu/flexible/snapshotter/config/Config.scala

```diff
@@ -4,11 +4,14 @@ import com.amazonaws.regions.{Region, Regions}
 import com.amazonaws.services.lambda.AWSLambdaClient
 import com.amazonaws.services.lambda.model.GetFunctionConfigurationRequest
 import com.amazonaws.services.lambda.runtime.Context
+import com.gu.flexible.snapshotter.Logging
 import play.api.libs.json.Json
 
 object Config {
-  def apiUrl(stage: String): String = stage match {
-    case "PROD" => "http://internal-Flexible-ApiLoadB-1QAHGRQLH03UW-649659201.eu-west-1.elb.amazonaws.com:8080"
+  def apiUrl(stage: String, stack: String): String = (stage, stack) match {
+    case ("PROD", "flexible")  => "http://internal-Flexible-ApiLoadB-1QAHGRQLH03UW-649659201.eu-west-1.elb.amazonaws.com:8080"
+    case ("PROD", "flexible-secondary") => "http://internal-Flexible-ApiLoadB-1NYF8NMK5QNMP-231637936.eu-west-1.elb.amazonaws.com:8080"
+    case ("CODE", "flexible-secondary") => "http://internal-Flexible-ApiLoadB-1LO2GC5WZ5S09-11639498.eu-west-1.elb.amazonaws.com:8080"
```

Comment:
I couldn't get it to work locally, I feel it should work too so will come chat when you're free.